tests/optimizers/conftest.py

```diff
@@ -13,6 +14,11 @@
 objective_name = "objective"
 
 
+@pytest.fixture
+def seed():
+    return random.getrandbits(32)
```

Comment:
I'm curious: What was your reason for using `random.getrandbits(32)` over `random.randint()`? (I didn't know `getrandbits` before and had to look it up)

Reply:
I guess the problem is that python integers are of arbitrary size and `random.randint()` needs to arguments (min and max) value. This way, we get an unsigned int with the usual 32bit size, which should work for all packages we use, i.e. numpy and torch.